pandas/io/excel.py

```diff
@@ -436,18 +602,182 @@ def _parse_cell(cell_contents,cell_typ):
                                 skip_footer=skip_footer,
                                 chunksize=chunksize,
                                 **kwds)
-            
+
             output[asheetname] = parser.read()
-            
+
         if ret_dict:
             return output
         else:
             return output[asheetname]
-        
+
+    def _parse_ods(self, sheetname=0, header=0, skiprows=None, skip_footer=0,
```

Comment:
so this should be a method on ``EZDFile`` called ``.parse``. (and the current ``._parse_excel`` should be on ``XLRDFile`` with the same name.